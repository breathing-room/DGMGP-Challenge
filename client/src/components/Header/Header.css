.header {
    /* background-color: #175c6d; */
    /* color: rgba(222, 253, 253); */
    padding: 8px;
    margin: 0 auto;
    margin-top: 10px;
    font-weight: 400;
}