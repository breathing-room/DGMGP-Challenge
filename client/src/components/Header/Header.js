import React from 'react';

const Header = () => {
    return (
        <h1>Don't Get Mad, Get the Weather</h1>
    )
}

export default Header;