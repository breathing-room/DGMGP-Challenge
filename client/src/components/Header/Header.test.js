import React from 'react';
import Header from './Header'

import { shallow } from 'enzyme';

describe('Header component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Header />)
    })

    test('it renders correctly', () => {   
      expect(wrapper).toMatchSnapshot();
    });

    test(' itrenders h5 element', () => {
        expect(wrapper.find('h5').text()).toBe(`Don't get mad, get the weather.`)
    })
})
