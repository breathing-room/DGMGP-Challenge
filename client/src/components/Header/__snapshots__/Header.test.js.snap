// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component it renders correctly 1`] = `ShallowWrapper {}`;
